apps/examples/src/examples/local-storage/LocalStorageExample.tsx

```diff
@@ -45,7 +38,7 @@ export default function PersistenceExample() {
 		const cleanupFn = store.listen(
 			throttle(() => {
 				const snapshot = store.getSnapshot()
-				setInLocalStorage(PERSISTENCE_KEY, JSON.stringify(snapshot))
+				localStorage.setItem(PERSISTENCE_KEY, JSON.stringify(snapshot))
```

Comment:
Are we keeping these calls?

Reply:
i figured we keep them in the examples app? we don't need to provide localstorage wrappers for consumers imo

Reply:
oh yea i was gonna make them all `@internal` as localstorage helpers don't seem like things we need to provide. but if u want us to have them as API surface lmk